src/reaper_osara.cpp

```diff
@@ -830,6 +830,10 @@ void postSetSelectionEnd(int command) {
 	fakeFocus = FOCUS_RULER;
 }
 
+void postToggleMasterMono(int command) {
+	outputMessage(GetToggleCommandState(command) ? "master track mono" : "master track stereo");
```

Comment:
Please change this to "master mono"/"master stereo", as I don't think track is needed to explain what you've pressed and we can thus safely reduce verbosity.